src/index/VocabularyMergerImpl.h

```diff
@@ -63,26 +65,28 @@ auto VocabularyMerger::mergeVocabulary(const std::string& basename,
     return lessThan(p1.entry_, p2.entry_);
   };
 
-  std::vector<cppcoro::generator<QueueWord>> generators;
+  // Open and prepare all infiles and mmap output vectors.
+  std::vector<ad_utility::InputRangeTypeErased<QueueWord>> generators;
+  generators.reserve(numFiles);
```

Comment:
Can you please split this change into a separate PR, 
As we yield single strings here I want to make sure, that we don't influence the performance here, 
and on the other hand, smaller PRs are also better.

Reply:
(I mean of course all the changes from this file)

Reply:
I will move this one to another pr.

Reply:
resolving this one as the splinter was merged